src/dataSources/WikibaseActionApi.js

```diff
@@ -13,26 +13,48 @@ module.exports = class WikibaseActionApi extends RESTDataSource {
     return this.getEntitiesLoader.load(id);
   }
 
+  async searchEntities(query, language) {
+    return await this.getSearchedEntities( {query, language} );
```

Comment:
Hm, this method just passes its arguments into `getSearchedEntities` and doesn't do anything on its own, so technically the contents of `getSearchedEntities` might as well move into this method.